internal/pkg/utils/slice.go

```diff
@@ -32,3 +32,12 @@ func SortByName(slice interface{}) interface{} {
 
 	return slice
 }
+
+func Contains[V comparable](slice []V, needle V) bool {
+	for _, v := range slice {
+		if v == needle {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Vo vseobecnosti je vzdy lepsie si spravit `map` a tu dalj pouzivat v podmienkach.
Samozrejme, pri `3` polozkach je rozdiel nemeratelny, ... ale zatial sa nam doteraz stale darilo tomuto kodu vyhybat. 

Problem nastane, ked to niekto pouzite bez rozmyslania na `1000+` hodnot, alebo v nejakej rekurzii.

Reply:
Preto Go taku funkciu nema: https://stackoverflow.com/a/10486196

Rovnako ako nema `.Map(), .Filter(), ...`, proste vsetky cykly a rekurzie maju byt v kode hned viditelne.

Reply:
Od `go1.18` existuje https://stackoverflow.com/a/71181131 `slices` pkg, ale ako pisem ^^^, neodporucam.